### ISSUE TITLE
Custom resources / models

### ISSUE THREAD

**Issue by ptheywood** [FLAMEGPU/FLAMEGPU2-visualiser#47]
Non stock resources are accessed by path, realtive to the working directory. 

Executables are not alwasy called from the same location, so this is limiting. Allowing a list of paths to search is a potential alternative? 

I.e. in an individual model repository (based on the template), if a resource is placed into `resources/cylinder_geom.obj`, this binary *must* be called from the repository root, rather than `build` (or any level deeper, or higher). 

Alternatively, a way of packing resources into the binary at the CMake level could be good. 

This willl probably need changes in both repos given how the vis headers are spread around.

**Comment by Robadob** [comment]
> Alternatively, a way of packing resources into the binary at the CMake level could be good.

Yeah this would be nice, should be possible to define a var, which is fed into the vis compilation's resource list. Albeit not 100% sure how at this point in time.

Probably not suitable for larger resources (e.g. if someone has a 200mb static model they want to include).
